exec/java-exec/src/main/java/org/apache/drill/exec/util/FileSystemUtil.java

```diff
@@ -42,6 +47,12 @@ public class FileSystemUtil {
 
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FileSystemUtil.class);
 
+  private static int recursiveListingMaxSize;
+
+  static {
+    recursiveListingMaxSize = DrillConfig.create().getInt(ExecConstants.RECURSIVE_FILE_LISTING_MAX_SIZE);
```

Comment:
This route to the config option felt pretty weird, I don't if there's a better way?